topostats/run_topostats.py

```diff
@@ -170,7 +170,10 @@ def run_topostats(args=None):
         else:
             summary_yaml = pkg_resources.open_text(__package__, "summary_config.yaml")
             summary_config = yaml.safe_load(summary_yaml.read())
-        summary_config = update_config(summary_config, args)
+
+        # Do not pass command line arguments to toposum as they clash with process command line arguments
```

Comment:
Not sure if this is a wider issue but have noted this in #634 just in case

Reply:
Thank you 😄